docs/products/kafka/index.rst

```diff
@@ -0,0 +1,48 @@
+Aiven for Apache Kafka
+======================
+
+What is Aiven for Apache Kafka?
+-------------------------------
+
+Aiven for Apache Kafka is a fully managed **distributed data streaming platform**, deployable in the cloud of your choice. Apache Kafka is an open source data streaming platform, ideal for event-driven applications, near-real-time data transfer and pipelines, stream analytics, and many more applications where a lot of data needs to move between applications in a speedy manner.
+
+Kafka stores a potentially large number of small data payloads, usually for a limited period of time. The storage is organised into "topics" and "partitions" so that many data streams can be handled at once, regardless of how many producers or consumers you need to work with.
+
+
+Why Apache Kafka?
+-----------------
+
+Apache Kafka itself is technically a distributed log storage mechanism; in reality it is a best-in-class, highly-available data streaming solution. Oh, and it just happens to have an incredibly rich ecosystem of open source tooling that connects to and extends the existing platform.
+
+Aiven for Apache Kafka MirrorMaker2
+'''''''''''''''''''''''''''''''''''
+
+By adding Aiven for Apache Kafka MirrorMaker2 to your setup, you gain replication superpowers. Whether you need data replication across clusters, availability zones or clouds, MirrorMaker2 is the answer.
+
```

Comment:
should we add "Aiven for Apache Kafka Connect" as well here?

Reply:
Ha, yes! I got distracted ...